internal/publish/config.go

```diff
@@ -55,20 +57,25 @@ type Config struct {
 	MaxMagnitudeSymptomOnsetDays uint          `env:"MAX_SYMPTOM_ONSET_DAYS, default=21"`
 	CreatedAtTruncateWindow      time.Duration `env:"TRUNCATE_WINDOW, default=1h"`
 
-	// Crypto key to use for wrapping/unwrapping the revision token cipher blocks.
-	RevisionTokenKeyID       string        `env:"REVISION_TOKEN_KEY_ID"`
-	RevisionTokenAAD         string        `env:"REVISION_TOKEN_AAD"` // must be base64 encoded, may come from secret://
 	RevisionKeyCacheDuration time.Duration `env:"REVISION_KEY_CACHE_DURATION, default=1m"`
-	RevisionTokenMinLength   uint          `env:"REVISION_TOKEN_MIN_LENGTH, default=28"`
 
 	// Flags for local development and testing. This will cause still valid keys
 	// to not be embargoed.
 	// Normally "still valid" keys can be accepted, but are embargoed.
 	ReleaseSameDayKeys bool `env:"DEBUG_RELEASE_SAME_DAY_KEYS"`
 }
 
+// RevisionTokenConfig represents the configuration and associated environment variables
+// for handling revision tokens.
+type RevisionTokenConfig struct {
+	// Crypto key to use for wrapping/unwrapping the revision token cipher blocks.
+	KeyID     string `env:"REVISION_TOKEN_KEY_ID"`
+	AAD       string `env:"REVISION_TOKEN_AAD"` // must be base64 encoded, may come from secret://
```

Comment:
We do [this](https://github.com/google/exposure-notifications-verification-server/blob/77c550f2db7d5a9c01ab0ec364f6027cec257a25/pkg/config/server_config.go#L140-L162) over in the verification server which will handle a base64-encoded byte slice. Then you can drop `RevisionTokenADDBytes()`.

Reply:
Otherwise you have to base64 decode it at runtime and worry about caching the result.

Reply:
(I realize this came from the existing thing above, but while we're moving it around...)

Reply:
Done!